lightningd/peer_control.c

```diff
@@ -355,6 +367,10 @@ void drop_to_chain(struct lightningd *ld, struct channel *channel,
 		log_broken(channel->log,
 			   "Cannot broadcast our commitment tx:"
 			   " they have a future one");
+	} else if (invalid_last_tx(channel->last_tx)) {
+		log_broken(channel->log,
```

Comment:
Now that I think more about it.... Should we not at least make an effort to send an `error` message to the peer, so that hopefully the peer can drop to chain for us?  Then maybe forget the channel?

Reply:
Yes. We've already sent an error to the peer at this point, which is why we're trying to broadcast the last state, fortunately.